WordPress/pattern-directory
# Add locale picker to pattern editor

This adds a locale field to the Pattern Details panel in the pattern editor (front and back end) The locale data that populates the field comes from the same mu-plugin used elsewhere on dotorg to supply locale data, so this also updates the local dev configuration to contain the required files and database table.

This uses postmeta to store the locale value rather than a taxonomy for the reasons described [here](https://github.com/WordPress/pattern-directory/issues/223#issuecomment-867874228).

See #223, #224

### Screenshots

![pattern-details](https://user-images.githubusercontent.com/916023/123352752-d97f6880-d514-11eb-86ea-1b2774b8e031.jpg)

### How to test the changes in this Pull Request:

1. Rebuild your local dev environment with `yarn run create`
2. Create or edit a pattern
3. Check the Pattern Details panel in the editor sidebar. The default language should be English, try choosing something else and saving, then reloading to ensure the value actually gets saved
4. Try using it from the New Pattern page and also from WP Admin
